recipes/d2/recipe.yaml

```diff
@@ -0,0 +1,51 @@
+context:
+  name: d2
+  version: 0.6.8
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - target_directory: src
+    url: https://github.com/terrastruct/d2/archive/refs/tags/v${{ version }}.tar.gz
+    sha256: 7b18f3f597a913912843fcd6eb52b926e343a2784b2f1009fce9e196fbf9ca03
+  - url: https://raw.githubusercontent.com/golang/freetype/refs/heads/master/LICENSE
+    sha256: d3ba056adc2b7909e95681deaae397fb37c97ed491a920f491214f07b62c41d0
+    file_name: freetype-licence.txt
+  - url: https://github.com/golang/freetype/raw/refs/heads/master/licenses/ftl.txt
+    sha256: f4b133e25df1f86ad3ffea453aa0e613f0474f34778dbbb3e437e7b2724937d8
+    file_name: freetype-licence-ftl.txt
+  - url: https://github.com/golang/freetype/raw/refs/heads/master/licenses/gpl.txt
+    sha256: c4120c6752c910c299e3bd9cb3a46ff262c268303ca2069b61f92f10a5656c18
+    file_name: freetype-licence-gpl.txt
+
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - ${{ compiler("go-nocgo") }}
+    - ${{ stdlib("c") }}
+    - go-licenses
+
+
+tests:
+  - script:
+    - d2 --help
+
+about:
+  homepage: https://github.com/terrastruct/d2
+  summary: D2 is a modern diagram scripting language that turns text to diagrams.
+  license: MPL-2.0
```

Comment:
Shouldn't you be putting an `AND GPL` here as you also list a GPL in the `license_file` section?

Reply:
https://raw.githubusercontent.com/golang/freetype/refs/heads/master/LICENSE freetype is `GPL-2.0-or-later OR LicenseRef-FreeType`, so this license should be `MPL-2.0 AND (GPL-2.0-or-later OR LicenseRef-FreeType)`.